subprojects/docs/src/docs/userguide/managingTransitiveDependencies.adoc

```diff
@@ -140,3 +147,23 @@ A project can decide to disable transitive dependency resolution completely. You
     <sourcefile file="build.gradle" snippet="transitive-per-configuration"/>
 </sample>
 ++++
+
+[[sec:bom_import]]
+=== Importing dependency constraints from Maven BOMs
```

Comment:
Could we maybe turn the title into something user would rather search for e.g. "Importing version recommendations from a Maven BOM"?

Reply:
Sure. You have better insights on that. I took your recommendation.